src/probeinterface/probe.py

```diff
@@ -600,14 +602,14 @@ def rotate_contacts(self, thetas):
         "_shank_ids",
     ]
 
-    def to_dict(self, array_as_list=False):
+    def to_dict(self, array_as_list: bool = False) -> dict:
```

Comment:
I would have never guessed that the type is bool here! 

`integer_as_float : bool` x D

Reply:
Oh, I see, the intent was something like `is_the_array_a_list`... now it is less confusing.